src/main/java/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+import duke.errors.DukeException;
+import duke.main.Storage;
+import duke.main.Ui;
+import duke.task.TaskList;
+
+/**
+ * Represents a command to exit the program. Command contains boolean to exit program.
+ */
+public class ExitCommand extends Command {
+
+    /**
+     * Executes the command
+     * @param tasks TaskList object that stores tasks
+     * @param ui Ui object deals with user interaction
+     * @param storage Storage object that handles text file
+     * @throws DukeException exception thrown in TaskList, Ui or Storage methods
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException {
+
+    }
+
+    /**
+     *
+     * @return true for program to exit
+     */
+    @Override
```

Comment:
I am not 100% sure but this isExit() method is very similar to that of the isExit() method in Command class, maybe instead of writing JavaDocs, you can use the @inheritDoc tag?